src/startupscriptgenerator/src/python/scriptgenerator.go

```diff
@@ -282,12 +282,25 @@ func (gen *PythonStartupScriptGenerator) buildGunicornCommandForModule(module st
 	}
 	
 	pythonEnableGunicornMultiWorkers := common.GetBooleanEnvironmentVariable(consts.PythonEnableGunicornMultiWorkersEnvVarName)
+
 	if pythonEnableGunicornMultiWorkers {
 		// 2N+1 number of workers is recommended by Gunicorn docs.
 		// Where N is the number of CPU threads.
 		// One worker will be reading or writing from the socket while the other worker is processing a request.
```

Comment:
nit: let's move this block comment about the `2N+1` into the `else` statement below where we set `workers` to that calculated value